src/Dockerfile

```diff
@@ -53,18 +58,12 @@ RUN cd /etc/.pihole && \
     install -Dm755 -t /usr/local/bin pihole && \
     install -Dm644 ./advanced/bash-completion/pihole /etc/bash_completion.d/pihole && \
     install -T -m 0755 ./advanced/Templates/pihole-FTL-prestart.sh /opt/pihole/pihole-FTL-prestart.sh && \
-    install -T -m 0755 ./advanced/Templates/pihole-FTL-poststop.sh /opt/pihole/pihole-FTL-poststop.sh
-
-
-ENV DNSMASQ_USER=pihole
-ENV FTL_CMD=no-daemon
-RUN addgroup -S pihole && adduser -S pihole -G pihole
-# RUN groupadd pihole && useradd -r --no-user-group -g pihole -s /usr/sbin/nologin pihole
-
-COPY bash_functions.sh /usr/bin/bash_functions.sh
-COPY start.sh /usr/bin/start.sh
+    install -T -m 0755 ./advanced/Templates/pihole-FTL-poststop.sh /opt/pihole/pihole-FTL-poststop.sh && \
+    # RUN groupadd pihole && useradd -r --no-user-group -g pihole -s /usr/sbin/nologin pihole
```

Comment:
```suggestion

```

This can go, too. It was a commented out line left over from some earlier experimentation